cdn/snippets.py

```diff
@@ -23,199 +23,195 @@
 
 import argparse
 import base64
-import datetime
+from datetime import datetime
 import hashlib
 import hmac
-
-from six.moves import urllib
+from urllib.parse import parse_qs, urlsplit
 
 
 # [START cloudcdn_sign_url]
-def sign_url(url, key_name, base64_key, expiration_time):
+def sign_url(
+    url: str, key_name: str, base64_key: str, expiration_time: datetime
+) -> str:
     """Gets the Signed URL string for the specified URL and configuration.
 
     Args:
-        url: URL to sign as a string.
-        key_name: name of the signing key as a string.
+        url: URL to sign.
+        key_name: name of the signing key.
         base64_key: signing key as a base64 encoded string.
-        expiration_time: expiration time as a UTC datetime object.
+        expiration_time: expiration time.
 
     Returns:
         Returns the Signed URL appended with the query parameters based on the
         specified configuration.
     """
     stripped_url = url.strip()
-    parsed_url = urllib.parse.urlsplit(stripped_url)
-    query_params = urllib.parse.parse_qs(
-        parsed_url.query, keep_blank_values=True)
-    epoch = datetime.datetime.utcfromtimestamp(0)
+    parsed_url = urlsplit(stripped_url)
+    query_params = parse_qs(parsed_url.query, keep_blank_values=True)
+    epoch = datetime.utcfromtimestamp(0)
     expiration_timestamp = int((expiration_time - epoch).total_seconds())
     decoded_key = base64.urlsafe_b64decode(base64_key)
 
-    url_pattern = u'{url}{separator}Expires={expires}&KeyName={key_name}'
+    url_to_sign = f"{stripped_url}{'&' if query_params else '?'}Expires={expiration_timestamp}&KeyName={key_name}"
 
-    url_to_sign = url_pattern.format(
-            url=stripped_url,
-            separator='&' if query_params else '?',
-            expires=expiration_timestamp,
-            key_name=key_name)
+    digest = hmac.new(decoded_key, url_to_sign.encode("utf-8"), hashlib.sha1).digest()
+    signature = base64.urlsafe_b64encode(digest).decode("utf-8")
 
-    digest = hmac.new(
-        decoded_key, url_to_sign.encode('utf-8'), hashlib.sha1).digest()
-    signature = base64.urlsafe_b64encode(digest).decode('utf-8')
+    return f"{url_to_sign}&Signature={signature}"
 
```

Comment:
Please remove empty lines before the closure of the region tag if `black`  formatter doesn't enforce them.